classes/locations.php

```diff
@@ -126,19 +129,19 @@ function register_location_taxonomy( $taxonomy, $tax_info ) {
 
 	// Add call back for meta box.
 	function location_meta_cb() {
-		add_meta_box( 'sm-location-premium-support', __( 'Premium Support', 'SimpleMap' ), array(
+		add_meta_box( 'sm-location-premium-support', __( 'Premium Support', 'simplemap' ), array(
 			&$this,
 			'premium_support',
 		), 'sm-location', 'side', 'high' );
-		add_meta_box( 'sm-geo-location', __( 'Geographic Location', 'SimpleMap' ), array(
+		add_meta_box( 'sm-geo-location', __( 'Geographic Location', 'simplemap' ), array(
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line indented incorrectly; expected at least 12 spaces, found 8
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->